docs/transfer/1-2-3-signed-authorization.puml

```diff
@@ -0,0 +1,48 @@
+@startuml
+
+title Transfer: 1.2.3 Signed Authorization
+
+
+box "PISP"
+participant "PISP Server" as D1
+end box
+box "Mojaloop"
+    participant Switch as S
+end box
+box "DFSP A"
+    participant "DFSP A\n(Payer)" as D2
+end box
+== Agreement Phase ==
+note right of D1
+  PISP checks the quote with the user,
```

Comment:
how PISP resolves `thirdpartyRequestsAuthorizationsId` to `thridpartyRequestsTransactionsId` so than can reroute the request to sign to proper device? PISP can have multiple transactions requests ongoing, it should be able to find a match between transaction request and authorization request

Reply:
Yep - I agree we need to revisit this.

Reply:
solved in above